config-template.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copy this file to config.js and fill it in with your own information.
+ */
+module.exports = {
```

Comment:
I guess I'm confused by what you mean. Does each part of the config need more info, or does the readme need to talk about it in more detail, or something else?